tests/unittests/QuantizationTest.cpp

```diff
@@ -368,17 +368,35 @@ TEST(Quantization, enableRowwiseQuantizedFullyConnected) {
   auto *S = F->createSave("ret", FC);
   bindings.allocate(S->getPlaceholder());
 
+  LoweredInfoMap loweredMapForQuant;
+  ::glow::lower(F, &loweredMapForQuant, EE.getBackend());
+
+  // Get the MatMul node and the Bactched_Add node.
+  Node *MatMul, *BatchedAdd;
+  for (Node &N : F->getNodes()) {
+    if (N.getKind() == Kinded::Kind::MatMulNodeKind) {
+      MatMul = &N;
+    }
+    if (N.getKind() == Kinded::Kind::BatchedAddNodeKind) {
+      BatchedAdd = &N;
+    }
+  }
+
```

Comment:
You should probably `ASSERT_TRUE()` on the pointers here to make sure they're correctly found.